test/ILLink.RoslynAnalyzer.Tests/DynamicallyAccessedMembersAnalyzerTests.cs

```diff
@@ -514,6 +514,7 @@ static string GetSystemTypeBase ()
 
 using System.Globalization;
 using System.Reflection;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
This should not be needed...

Reply:
This is needed for the `DynamicallyAccessedMembersAttribute`s below in the `TestSystemTypeBase` class in the string.